tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/ompl/impl/config/ompl_planner_freespace_config.hpp

```diff
@@ -43,15 +43,15 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 namespace tesseract_motion_planners
 {
 /** @brief Construct a basic planner */
-template <typename PlannerType>
-OMPLPlannerFreespaceConfig<PlannerType>::OMPLPlannerFreespaceConfig(tesseract::Tesseract::ConstPtr tesseract_,
-                                                                    std::string manipulator_)
-  : OMPLPlannerConfig<PlannerType>(std::move(tesseract_), std::move(manipulator_))
+template <typename... PlannerType>
```

Comment:
Is this a remnant of the previous effort? I don't think this class is templated any more, and this file should probably be moved to a .cpp source file

Reply:
Yep Fixed